manifests/d/Daum/PotPlayer/230830/Daum.PotPlayer.locale.en-US.yaml

```
@@ -1,8 +1,8 @@
-# Created with Komac v1.11.0
+# Created using wingetcreate 1.5.2.0
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
 
 PackageIdentifier: Daum.PotPlayer
-PackageVersion: '230830'
+PackageVersion: "230830"

```

```suggestion
PackageVersion: "230905"
```
This should match control panel.